docs/component-spec/background-linters.md

```diff
@@ -8,44 +8,45 @@ redirect_from: "/docs/developer-guide/background-linters/"
 
 # Setting up background linters
 
-ESLint *should* be used when developing Origami components. It's useful to have linters run in the background and highlight errors as you're typing. SublimeLinter and ESLint packages are recommended for Sublime Text and JetBrains IDEs respectively.
+ESLint and SASS Lint *should* be used when developing Origami components. It's useful to have linters run in the background and highlight errors as you're typing.
 
-## Install dependencies
+If you already have Origami Build Tools (OBT) with ESLint and SASS Lint configured, skip to the [Installing background linter plugins](#installing-background-linter-plugins) section.
 
-If you already have Origami Build Tools (OBT) and ESLint configured, skip to the [Installing background linter plugins](#installing-background-linter-plugins) section.
+## Install dependencies
 
-Ensure you have installed [Origami Build Tools](https://www.npmjs.com/package/origami-build-tools). This will provide you with Sass, scss-lint and provide instructions on how to set up Node.js and Ruby.
+Ensure you have installed [Origami Build Tools](https://www.npmjs.com/package/origami-build-tools). This will provide you with [SASS Lint](https://github.com/sasstools/sass-lint) and [ESLint](https://github.com/eslint/eslint).
 
-### Install ESLint
+## Configure
 
-ESLint isn't installed automatically installed as part of OBT, so install it with the following command:
+You can have an `.eslintrc` configuration file per folder. It is recommended to use the [Origami global configuration](https://github.com/Financial-Times/origami-build-tools/blob/master/config/.eslintrc) when developing Origami components. We recommend installing the configuration file into your home directory (User directory) to avoid installing it into each individual project folder.
```

Comment:
Remove passive voice and replace "an" with "one" which... I'm not sure why but I think it reads better? 

You can have ~an~ *one* `.eslintrc` configuration file per folder. ~It is recommended to~ *We recommend that you* use the [Origami global configuration](https://github.com/Financial-Times/origami-build-tools/blob/master/config/.eslintrc) when developing Origami components.